app/models/concerns/courses/edit_options/study_mode_options.rb

```diff
@@ -0,0 +1,12 @@
+module Courses
+  module EditOptions
+    module StudyModeOptions
```

Comment:
What do people think about the naming? My feeling is that this should be `StudyModeOptionsConcern` ... that's the standard for Rails classes/modules that aren't models, and even though this is a concern for a model I think that this qualifies.

I know we haven't done this for the existing concerns, or the PageObjects for that matter, I'd be happy to see a commit that's just designed to rename them all instead of doing them piecemeal.

Reply:
> My feeling is that this should be `StudyModeOptionsConcern`

Actually, the full path should be something like `Courses::EditOptions::StudyModeConcern` ... no need to repeat `Options` imo.

Reply:
👍 I'd like to see `Concern` suffix being used. 

But I also think the 3 levels of nesting with two `Options` could be improved upon too. 

As how would you do it?  E.g. `module CoursesConcerns::StudyModeOptionsConcern`

I don't like specific concern just for `Edit`

Reply:
I don't think much of it, I think current is fine and the suggested one with the `Concern` suffix is better but I don't have strong feelings for changing this now :)

Reply:
Udpated them all 👍